### PR TITLE
adasdasd

### PR DESCRIPTION
**Pull request by Misfiy**

### CODE REVIEW

File: SillySCP/API/IInittable.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿namespace SillySCP.API
+{
+    public interface IInittable
```
Comment (moddedmcplayer) [inline SillySCP/API/IInittable.cs:3]: IRegisterable?

Comment (Misfiy) [reply SillySCP/API/IInittable.cs:3]: probably a better name yea, i put a temp one as an example
its a draft PR

Comment (JayXTQ) [reply SillySCP/API/IInittable.cs]: Changed to IRegisterable @moddedmcplayer

**Review by Misfiy — COMMENTED** [review]
lgtm

**Pull request merged**